docs/python/tutorial-django.md

```diff
@@ -351,13 +355,15 @@ In this section you start by creating a single page using a template. In subsequ
     ```html
     <!DOCTYPE html>
     <html>
-        <head>
```

Comment:
Revert this change to the template: the code is this way on purpose, to introduce tags in the template in the simplest form first, and to show why you don't want to build HTML up in code. Furthermore, you've suddenly dropped in static files from a later part of the tutorial, so I'm guessing you made this change mistakenly.

Reply:
Good eye! I didn't mean to remove this and reverted the change.